### PR TITLE
Aws roles

### PR DESCRIPTION
**Pull request by Kolezhanchik**
Aws Roles dashboard and functionality 
image
image

image

### CODE REVIEW

File: app/(product)/public-cloud/(product)/aws-roles/[licencePlate]/[role]/page.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  if (!res.ok) {
+    throw new Error('Network response was not ok for fetch user image');
+  }
+  const data = await res.json();
+  return data;
+}
+
+export default function ProductAWSRoles() {
+  const pathName = usePathname();
+  const { licencePlate, userRole, currentPage, pageSize } = parseUrlParams(pathName);
```
Comment (junminahn) [inline app/(product)/public-cloud/(product)/aws-roles/[licencePlate]/[role]/page.tsx:44]: What don't we use `useParams` to get `path params` and use `useSearchParams` to get `query params` in client side?
- https://nextjs.org/docs/app/api-reference/functions/use-params
- https://nextjs.org/docs/app/api-reference/functions/use-search-params

Also, take a look at the helper function `parsePaginationParams` in the our codebase:
- https://github.com/bcgov/platform-services-registry/blob/main/app/zap/results/page.tsx#L141

**Review by junminahn — COMMENTED** [review]
As agreed this morning, can you identify the `tab(s)` and hide them depends on the environment variable `APP_ENV`?

File: app/(product)/public-cloud/(product)/aws-roles/[licencePlate]/[role]/page.tsx (hunk 1 above)
Comment (Kolezhanchik) [reply app/(product)/public-cloud/(product)/aws-roles/[licencePlate]/[role]/page.tsx]: changed, thank you!

File: app/(product)/public-cloud/(product)/layout.tsx [hunk 2]
```diff
@@ -16,7 +17,7 @@ const tabsData = [
     label: 'ROLES',
     name: 'aws-roles',
   },
-];
+].filter((tab) => (process.env.APP_ENV === 'prod' ? (tab.label === 'ROLES' ? null : tab) : tab));
```
[the remark this answers is not in the document]
Comment (Kolezhanchik) [reply app/(product)/public-cloud/(product)/layout.tsx]: changed, thank you!

[the remark this answers is not in the document]
Comment (Kolezhanchik) [reply app/(product)/public-cloud/(product)/layout.tsx]: yes, you are right, my bad, changed

**Review by junminahn — APPROVED** [review]

**Pull request merged**